### PR TITLE
PMM-9652 Fixed anti-pattern metric names

### PR DESCRIPTION
**Pull request by percona-csalguero**
[PMM-9652](https://jira.percona.com/browse/PMM-9652) DB Stat and RealTime MongoDB dashboards

This PR is part of the solution for PMM-9652 and implements the changes requested in https://github.com/percona/mongodb_exporter/issues/394

### CODE REVIEW

File: exporter/collstats_collector.go [hunk 1]
```diff
@@ -101,11 +101,10 @@ func (d *collstatsCollector) Collect(ch chan<- prometheus.Metric) {
 		// Since all collections will have the same fields, we need to use a metric prefix (db+col)
 		// to differentiate metrics between collection. Labels are being set only to matke it easier
 		// to filter
-		prefix := database + "." + collection
+		prefix := "collstats"
 
 		labels := d.topologyInfo.baseLabels()
-		labels["database"] = database
-		labels["collection"] = collection
+		labels["namespace"] = database + "." + collection
```
Comment (rnovikovP) [inline exporter/collstats_collector.go:107]: Considering  https://prometheus.io/docs/practices/instrumentation/#things-to-watch-out-for and  https://prometheus.io/docs/practices/naming/
I woud recommend to keep DB and Collectoin separated

Comment (percona-csalguero) [reply exporter/collstats_collector.go]: :+1:

File: exporter/collstats_collector.go [hunk 2]
```diff
@@ -101,11 +101,10 @@ func (d *collstatsCollector) Collect(ch chan<- prometheus.Metric) {
 		// Since all collections will have the same fields, we need to use a metric prefix (db+col)
 		// to differentiate metrics between collection. Labels are being set only to matke it easier
 		// to filter
```
[the remark this answers is not in the document]
Comment (percona-csalguero) [reply exporter/collstats_collector.go]: :+1:

**Review by BupycHuk — APPROVED** [review]

File: exporter/top_collector.go [hunk 3]
```diff
@@ -33,6 +35,12 @@ type topCollector struct {
 	topologyInfo   labelsGetter
 }
 
+var ErrInvalidOrMissingTotalsEntry = fmt.Errorf("Invalid or misssing totals entry in top results")
+
+type topResponse struct {
```
Comment (ShashankSinha252) [inline exporter/top_collector.go:40]: Unused type?

**Review by ShashankSinha252 — COMMENTED** [review]
LGTM. Just 1 query, posted in form of review comment.

**Pull request merged**